### DISCUSSION TITLE
cpptools version1.13.7 is not  worked under vscode version 1.74.0?

### DISCUSSION THREAD

**Discussion by dalianzhi** [microsoft/vscode-cpptools#10277] [Q&A]
cpptools version1.13.7 is not  worked under vscode version 1.74.0?Do not display in the extension column.

**Comment by michelleangela** [comment]
@dalianzhi 
Which OS and version are you using? Could you try un-installing the extension manually by deleting it from the [extension's folder](https://code.visualstudio.com/docs/editor/extension-marketplace#_where-are-extensions-installed) and re-installing it?
